sonar/modules/users/api.py

```diff
@@ -281,21 +281,17 @@ def get_moderators_emails(self):
 
     def is_granted(self, role_to_check):
         """Check if user has at least the role passed in argument."""
```

Comment:
No, `at least` is still valid, because the method return true if `role_to_check` is `moderator` and user role is `admin`, because of the role hierarchy.